src/warnings.cpp

```diff
@@ -51,36 +54,39 @@ bool GetfLargeWorkInvalidChainFound()
     return fLargeWorkInvalidChainFound;
 }
 
-std::string GetWarnings(const std::string& strFor)
+std::pair<std::string, int64_t> GetWarnings(const std::string& strFor)
 {
+    std::pair<std::string, int64_t> rpc;
+    std::pair<std::string, int64_t> statusbar;
+    rpc.second = GetTime();
+    statusbar.second = GetTime();
```

Comment:
Nit: `GetTime()` can advance (or change completely) in between these calls; instead try:
```suggestion
    statusbar.second = rpc.second = GetTime();
```